core/EE_Config.core.php

```diff
@@ -2332,6 +2332,13 @@ class EE_Registration_Config extends EE_Config_Base
      */
     public $gateway_log_lifespan = '1 week';
 
+    /**
+     * Enable copy attendee info at form
+     *
+     * @var boolean $enable_copy_attendee
+     */
+    public $copy_attendee_info;
```

Comment:
These config classes were originally setup with `public` properties but that was a bad choice as it exposes the internals of the class.

For any new properties that we add, we want to not have them `public`, and instead use `private` or `protected` (you likely need to use `protected` if you want to mock the class for testing purposes) and then use public setters and getters for controlling access. My apologies for not explaining this to you earlier. My bad.

A good example that you can follow is the `EE_Registration_Config->track_invalid_checkout_access` property. 

As well, the config classes do not use the model system like our entities do (like events, etc) and so their properties need to be sanitized (another super important reason to use a setter). Since this new property you are adding is a boolean, you can use the PHP `filter_var()` function to sanitize it. Again, see the setter for `track_invalid_checkout_access` as an example.

This also means that you will need to change the places that retrieve this value to use the new getter instead of directly accessing the property.

As well, for any new code we want to follow the [PSR guidelines](https://www.php-fig.org/psr/psr-2/) for formatting and styles, so any new methods you add should use camelCase for the naming pattern.